src/datepicker/datepicker.component.ts

```diff
@@ -284,6 +283,12 @@ export class DatePicker implements OnDestroy, OnChanges, AfterViewChecked {
 		const daysContainer = document.querySelectorAll(".flatpickr-days");
 		const dayContainer = document.querySelectorAll(".flatpickr-day");
 
+		Array.from(calendarContainer)
```

Comment:
This doesn't belong to `updateClassNames`. It's not doing anything to class names and it's not necessary to update any class names.
Either function needs a better name or this needs to be moved out to a different function and used in a different way. Since it's a protected function and not used anywhere else, former might make more sense.

Reply:
That makes sense